python/paddle/v2/framework/tests/test_exception.py

```diff
@@ -0,0 +1,12 @@
+import paddle.v2.framework.core as core
+import unittest
+
+
+class TestException(unittest.TestCase):
+    def test_exception(self):
+        self.assertRaises(core.EnforceNotMet,
+                          lambda: core.__unittest_throw_exception__())
```

Comment:
How to get and check the error message? Use `.message()`?
It will be better to add another unit test of the error message here.

Reply:
Done.